### ISSUE TITLE
Call to a member function getEventManager() on null in OnBootstrapListener

### ISSUE THREAD

**Issue by leogr** [zendframework/zend-modulemanager#11]
With zendframework/zend-modulemanager 2.5.2:
```
Fatal error: Call to a member function getEventManager() on null in /var/www/webapu/vendor/zendframework/zend-modulemanager/src/Listener/OnBootstrapListener.php on line 35
```

**Comment by grizzm0** [comment]
This has just been fixed by @weierophinney and 2.5.3 has been pushed. Please run composer update again (you might need to run composer clear-cache)

**Comment by leogr** [comment]
Now it works! :+1:
